.github/actions/save/action.yml

```
@@ -2,20 +2,26 @@ name: save
 description: Save a directory to a cache based on a shared directory
 inputs:
   root:
-    description:
+    description: Directory for cache
     default: /cache
   path:
     description: Directory to save to a cache
     required: true
   dest:
     description: Directory in cache
     required: true
+  enabled:
+    description: Enable cache
+    default: "true"
 runs:
   using: "composite"
   steps:
     - name: Save ${{ inputs.path }} to cache
```

Why not something like that?
```suggestion
    - name: Save ${{ inputs.path }} to cache
      if: ${{ inputs.enabled == 'true' }}
```

---

You are right, it's simpler. I'll make a change.